Makefile

```
@@ -1,5 +1,13 @@
 all: bin/dver bin/dreg
 
+TAG=core-dreg:latest
+
+docker-image:
+	docker build . -t $(TAG)
```

Put `-t` option first:

```suggestion
	docker build -t $(TAG) .
```

> **Usage**
> ```
>  docker build [OPTIONS] PATH | URL | -
> ```
>
> https://docs.docker.com/engine/reference/commandline/build/